### DISCUSSION TITLE
Add support for integrating extent reports with Codeception

### DISCUSSION THREAD

**Discussion by samiullah** [Codeception/Codeception#6187] [Ideas]
#### What are you trying to achieve?
I am trying to  use extent reports for reporting
#### What do you get instead?
Was not able to find any way to do it

> Provide console output if related. Use `-vvv` mode for more details.

```bash
# paste output here
```
> Provide test source code if related

```php
// paste test
```
### Details

* Codeception version: 
* PHP Version:
* Operating System:
* Installation type: Phar || Composer
* List of installed packages (`composer show`)
* Suite configuration:

```yml
# paste suite config here
```

**Comment by DavertMik** [comment]
That's a good idea but could you explain it a bit further?

**Comment by samiullah** [comment]
@DavertMik  Though we have html reports available in codeception and screenshots can be seen on failure. In extent reports we can have screenshots at every step wherever we want and charts for reporting are much readable.
